Gopkg.lock

```diff
@@ -10,6 +24,9 @@
 [[projects]]
   name = "github.com/Azure/azure-sdk-for-go"
   packages = [
+    "arm/authorization",
```

Comment:
Could we please use either of these:

* `github.com/Azure/azure-sdk-for-go/services/authorization/mgmt/2015-07-01/authorization`
* `github.com/Azure/azure-sdk-for-go/services/authorization/mgmt/2017-10-01-preview/authorization`

The `arm/` paths won't work in 14+.

Reply:
This is a substantial change and outside the scope of this PR. It must be fixed by whoever is responsible for this sample, not me.

Reply:
I am fixing this :)

Reply:
Thank you! 👍

Reply:
+1, thanks @mcardosos!